pkg/api/org_users.go

```diff
@@ -323,12 +323,13 @@ func (hs *HTTPServer) searchOrgUsersHelper(c *contextmodel.ReqContext, query *or
 
 	// Get accesscontrol metadata and IPD labels for users in the target org
 	accessControlMetadata := map[string]accesscontrol.Metadata{}
-	if c.QueryBool("accesscontrol") && c.SignedInUser.Permissions != nil {
-		// TODO https://github.com/grafana/identity-access-team/issues/268 - user access control service for fetching permissions from another organization
-		permissions, ok := c.SignedInUser.Permissions[query.OrgID]
-		if ok {
-			accessControlMetadata = accesscontrol.GetResourcesMetadata(c.Req.Context(), permissions, "users:id:", userIDs)
+	if c.QueryBool("accesscontrol") {
+		permissionsList, err := hs.accesscontrolService.GetUserPermissions(c.Req.Context(), c.SignedInUser, accesscontrol.Options{OrgID: &query.OrgID})
```

Comment:
is it ok to not rely on any caching here?

Not blocking

Reply:
What do you mean? `GetUserPermissions()` uses cache if it's enabled. Do you suggest to disable it here?